### PR TITLE
Make role work on Rocky9 and Ubuntu 22.04

### PR DESCRIPTION
**Pull request by pwalczysko**
As this role is used in the playbook for demoserver, this PR is attempting to make it Rocky9 compatible.

@jburel @sbesson @khaledk2

### CODE REVIEW

**Comment by pwalczysko** [comment]
The tests are passing on https://github.com/pwalczysko/ansible-role-omero-user/actions/runs/7504778194

**Comment by pwalczysko** [comment]
Tests are passing here too :)

**Review by sbesson — COMMENTED** [review]
As far as I can tell there are no functional changes right, only testing and metadata updates?

The deployment step in the GitHub workflow might also need to be adjusted to use @jburel's latest changes.

**Review by sbesson — APPROVED** [review]
Looks good to me but I'll leave @jburel to comment on the syntax of the `publish` step

File: .github/workflows/molecule.yml [hunk 1]
```diff
@@ -46,10 +46,17 @@ jobs:
...
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
-      - name: galaxy
-        uses: robertdebock/galaxy-action@1.2.1
+      - uses: actions/checkout@v4
+      - name: Read the role name
+        id: role-name
+        run: |
+          name=$(grep 'role_name' meta/main.yml | sed -r 's/^[^:]*:(.*)$/\1/' | tr -d '[:space:]')
```
Comment (jburel) [inline .github/workflows/molecule.yml]: I ended having to split the line
see https://github.com/ome/ansible-role-nfs-share/pull/8/commits/3ed6b1ef404414f9b3f4372b5670de51e1583091

**Review by jburel — APPROVED** [review]

**Pull request merged**